test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -218,17 +222,17 @@ func describeQatDpdkPlugin() {
 	})
 }
 
-func runCpaSampleCode(ctx context.Context, f *framework.Framework, runTests int, resourceName v1.ResourceName) {
-	ginkgo.By("submitting a pod requesting QAT" + resourceName.String() + "resources")
+func runQatEngineTestAppOrCpaSampleCode(ctx context.Context, f *framework.Framework, command []string, resourceName v1.ResourceName) {
```

Comment:
We may want to re-think this function completely. I know it follows `runTestCase()` from FPGA e2e. I think what we would want here is more like `createPod()` and the rest could move to `It()` nodes. This would make things aligned with that the DPDK test nodes are doing.

Reply:
I see your point.!
Would it be better, then, to make `createPod` of fpga to be in `utils.go` and use it in both qat and fpga?

Reply:
The idea is ok but could be done later. It looks good to me as it is now.